drivers/include/mtd_flashpage.h

```diff
@@ -59,6 +59,7 @@ extern const mtd_desc_t mtd_flashpage_driver;
  */
 typedef struct {
     mtd_dev_t base;     /**< MTD generic device */
+    size_t offset;      /**< offset (in pages) from the start of the flash */
```

Comment:
In the past, I used `mtd_mapper` for this purpose. Maybe we can use it here, too and save some LOC in this driver? Just an idea ...

Reply:
My concern was that `mtd_mapper` would pull in a lot more LOC and adding a static offset would be much simpler.

My rationale was that if you want to use the internal flash as MTD device, you always need an offset to not overwrite the firmware, so might as well add this here.